ansible/monitoring.yml

```diff
@@ -44,10 +44,14 @@
     datadog_api_key: "{{ vault_datadog_api_key|default('DATADOG_API_KEY') }}"
     datadog_agent6: true
     datadog_config:
```

Comment:
I see from ansible/requirements.yml that datadog plugin is https://galaxy.ansible.com/datadog/datadog v 2.1.0
so this var is identified [here](https://github.com/DataDog/ansible-datadog/blame/2.1.0/README.md#L28).

Reply:
Not clears to me who installs the datadog agent for processing this config.

Reply:
The datadog role here: https://github.com/aeternity/infrastructure/pull/108/files/be88057b106813d5e2754fc847f2a52ea84a16a1#diff-f1b92d458d2435cc2d62490c42d86397R128

Reply:
[`datadog` pip](https://pypi.org/project/datadog/) is installed [here](https://github.com/aeternity/infrastructure/blob/1dbc21209d65bc439330b0cb28eedca3b4d16aa2/ansible/setup.yml#L110).

Reply:
If you meant the role itself, it's installed with this file: https://github.com/aeternity/infrastructure/blob/master/ansible/requirements.yml#L1
by https://github.com/aeternity/infrastructure/blob/master/terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash#L13